doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -673,6 +694,23 @@ Make sure that it is enabled for the following use cases of the Google Fast Pair
 
 See :ref:`ug_bt_fast_pair_use_case` for more details about the use cases of the Google Fast Pair application.
 
+.. _ug_bt_fast_pair_gatt_service_firmware_update_intent:
+
+Firmware update intent
```

Comment:
Can you think of a bit different name as you already have a section with the exactly same name earlier in the doc?

Reply:
We used the same approach for the "FMDN extension" section name that appears 6 times here. The idea is that certain features require certain actions at each integration steps:

Step 1 -> Configuration of the Google Nearby Device console
Step 2 -> Configuration of the GATT service in the firmware 
...
Step x ->

The user is supposed to go through all integration steps and make sure that they follow all the guidelines if they are interested in certain feature (e.g. FMDN extension. firmware update intent).